lib/hets_agent_invoker.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# A service class that finds the new files of a commit and invokes the HetsAgent
+# to analyze them.
+class HetsAgentInvoker
+  attr_reader :commit_sha, :connection, :git, :repository
+
+  def initialize(repository_id, commit_sha)
+    @commit_sha = commit_sha
+    @repository = RepositoryCompound.find(id: repository_id)
+    @git = @repository.git
+    @connection = Bunny.new
```

Comment:
Should use `Sneakers::CONFIG[:connection]`

Reply:
Fixed it (only this line) by commit amend.